package/kernel/mac80211/patches/ath11k/906-ath11k-disable-coldboot-calibration-for-ipq6018.patch

```diff
@@ -0,0 +1,13 @@
+--- a/drivers/net/wireless/ath/ath11k/core.c
```

Comment:
Please export this with `git format-patch` with a proper title, description, and SoB.